src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -489,4 +517,39 @@ private void EmitClassVariablesAndConstructor(RestApiType restApiType, string ht
         OutLn($"{optionsName} = autoClientOptions;");
         OutCloseBrace();
     }
+
+    private void EmitPathParameterValidation(string staticsName)
+    {
+        OutLn($$"""
+            private static bool IsPathParameterValid(string value)
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                {
+                    return false;
+                }
+
+                if (value.IndexOfAny({{staticsName}}.Slashes) > -1)
```

Comment:
If we assume that 99% of the times, the strings will not contain slashes or dots, can we make a fast trivial exit paths to reduce computation?

if (value.IndexOfAny(SlahesOrDots) < 0)
   return true;

Or something like it.

Reply:
Done